src/resolve.ts

```diff
@@ -63,11 +63,8 @@ export function jitiResolve(
     if (resolved) {
       return resolved;
     }
-    // Try resolving .ts files with .js extension
-    if (
-      TS_EXT_RE.test(ctx.filename) ||
-      TS_EXT_RE.test(ctx.parentModule?.filename || "")
```

Comment:
While tests might pass (and I immediately cannot recall why we needed this check) it might be a safer option to add new checks in addition to the current ones.

Reply:
Yeah just to make you aware of the situation. Pushed a minimal change/fix now.